### ISSUE TITLE
nodemone stuck on "restarting due to changes..." when working with java package

### ISSUE THREAD

**Issue by rrpereira** [remy/nodemon#1865]
- `nodemon -v`: 2.0.7
- `node -v`: v14.17.0
- Operating system/terminal environment: linux
- Using Docker? no
- Command you ran: nodemon test.js

### Expected behaviour

If i run test.js as follows, it won't have any problem because line 8 (where i am importing a java library) is commented.

![Selection_008](https://user-images.githubusercontent.com/36520545/122188370-cb33ab80-ce87-11eb-83a8-fcfb51f1dbd0.png)

### Actual behaviour

If i uncomment line 8 (which is how it should be) and if i run "nodemon test.js", at first it will work perfectly, but after adding a console.log('hello') and save test.js, for example, it will get stuck at "[nodemon] restarting due to changes...". This problem appears with any other change made within test.js.

![Selection_007](https://user-images.githubusercontent.com/36520545/122187571-0da8b880-ce87-11eb-83c0-ee7b15f82872.png)

### Steps to reproduce

Execute "nodemon test.js", change something (in my case i added a console.log('problem') at line 9), save test.js and the server gets stuck: 

![Selection_010](https://user-images.githubusercontent.com/36520545/122190765-0636de80-ce8a-11eb-8297-5f9cbdf0ed41.png)

**Comment by joshuasorkin** [comment]
I am having the same problem in Windows 10, Node v14.16.1.

**Comment by roniceyemeli** [comment]
same issue here

**Comment by roniceyemeli** [comment]
> I am having the same problem in Windows 10, Node v14.16.1.

^hey, i used node-supervisor instead of nodemon and it's working for me!

**Comment by remy** [comment]
@joshuasorkin you're using java with nodemon too?

@rrpereira can you try with a forced spawn, using `nodemon --spawn test.js` - I suspect under the hood (of your code) when you spawn out Java it's doing something fun with the process hooks.

**Comment by DakouriKobri** [comment]
Using PowerShell instead of Bash on my Windows 10 helped me fixed the issue.

I had almost the same issue whereby changes to my js file could not be seen at my port and nodemon got stuck with the restarting message as in the above original post. I initially created a server with Node.js and Express.js (server.js). I installed Nodemon and could start my server with

`nodemon server.js`

However, I did all this using Bash. The issue got sorted out when I moved from Bash to PowerShell.
